recaf-core/src/main/java/me/coley/recaf/workspace/resource/source/MavenContentSource.java

```diff
@@ -0,0 +1,102 @@
+package me.coley.recaf.workspace.resource.source;
+
+import me.coley.recaf.workspace.resource.Resource;
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static java.io.File.separatorChar;
+import static java.lang.String.format;
+
+/**
+ * Origin location information for maven artifacts.
+ *
+ * @author Matt Coley
+ */
+public class MavenContentSource extends ContentSource {
+	private static final String CENTRAL_REPO = "https://repo1.maven.org/maven2";
```

Comment:
I think it's a good idea to make this url customizeable.

Reply:
It can now be passed via the constructor